deep-learning/_posts/2017-05-17-inception.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: review
+title:  "Inception-v3 : Rethinking the Inception Architecture for Computer Vision"
+tags:   deep-learning CNN classification
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1512.00567.pdf"
+
+cite:
+  authors: "C.Szegedy, V.Vanhoucke, S.Ioffe, J.Shlens"
+  title:   "Rethinking the inception architecture for computer vision"
+  venue:   "roceedings of the IEEE Conference on Computer Vision and Pattern Recognition (pp. 2818-2826)"
+---
+
+
+The main contribution of this paper is the **factorized convolutions**. In *Inception-v1*, the authors use bigger kernel size in their architecture, which are computationally expensive.
+
+In *Inception-v2*, the authors propose to factorize all kernels bigger than 3 to multiple *3x3* kernels. So a *5x5* kernel would become two *3x3*.
+
+<div align="middle">
+  <img src="/deep-learning/images/inception/fig_5.png" width="400">
```

Comment:
This image represents the inception module used in v2 and v3. v1 uses bigger kernel sizes (see 2nd sentence)